.cspell.dict/rust-more.txt

```diff
@@ -10,6 +10,7 @@ byteorder
 byteset
 caseless
 chrono
+Chunker
```

Comment:
This is looking more like python term or RustPython original term rather than coming from Rust environment.

Reply:
```suggestion
```